geode-core/src/test/java/org/apache/geode/management/internal/api/LocatorClusterManagementServiceTest.java

```diff
@@ -125,16 +127,41 @@ public void before() throws Exception {
     doReturn(true).when(persistenceService).lockSharedConfiguration();
     doNothing().when(persistenceService).unlockSharedConfiguration();
     operationManager = mock(OperationManager.class);
+    dLockService = mock(DistributedLockService.class);
+
     service =
         spy(new LocatorClusterManagementService(cache, persistenceService, managers, validators,
             memberValidator, cacheElementValidator, operationManager));
+    doReturn(dLockService).when(service).getCmsDlockService();
 
     regionConfig = new Region();
     regionConfig.setName("region1");
 
     rebalanceOperation = new RebalanceOperation();
   }
 
+  @Test
+  public void lockAndUnlockCalledAtCreate() {
+    try {
+      service.create(regionConfig);
+    } catch (Exception ignore) {
```

Comment:
Is create expected to throw some Exception every time this test runs? If not, then let's remove the try-catch and add `throws Exception` clause to the method.

Reply:
same as "delete" case.